desktop/api/src/main/java/org/datacleaner/util/WidgetFactory.java

```diff
@@ -350,6 +351,17 @@ public static JXTaskPane createTaskPane(final String title, final Icon icon) {
         return taskPane;
     }
 
+    public static JXTitledPanel createTitledPanel(final String title, final JComponent content) {
+        final JXTitledPanel titlePanel = new JXTitledPanel(title, content);
+        titlePanel.setFocusable(false);
+        titlePanel.setTitlePainter((graphics2D, panel, width, height) -> {
```

Comment:
Can we move the titlePainter configuration into the LookAndFeelManager, so it's a little better separated, or does the UIManager also work terribly for "JXTitledPanel.titlePainer"?

Otherwise it's no biggie, we're doing plenty of horrible things in here anyway, but it would be nice not to do even more. :)